views/order-modify.html.twig

```diff
@@ -0,0 +1,93 @@
+{% extends 'base.html.twig' %}
+{% block title %}Order - Cancel - {% endblock %}
+{% block body %}
+    <h2 class="{% if type == 'cancel' %} text-danger {% elseif type == 'restore' %} text-success {% else %} text-primary {% endif %}">
```

Comment:
I think the default (else condition) text should not have any class (i.e. default black color).  The `text-primary` style is confusing because it appears as if the title is a link.

Reply:
Here the default class `text-primary` is for unlock. Yes, it's bit confusing that it looks like a link may be we should think of a different custom color for unlock? Will remove the `text-primary` for now.

Reply:
I think no custom color is fine.